src/db/sysdb_sudo.c

```diff
@@ -480,7 +480,8 @@ sysdb_get_sudo_user_info(TALLOC_CTX *mem_ctx,
                     sss_get_cased_name(sysdb_groupnames, groupname,
                                        domain->case_sensitive);
                 if (sysdb_groupnames[num_groups] == NULL) {
-                    DEBUG(SSSDBG_MINOR_FAILURE, "Cannot strdup %s\n", groupname);
+                    DEBUG(SSSDBG_CRIT_FAILURE,
+                          "sss_get_cased_name() failed for '%s'\n", groupname);
```

Comment:
I am wondering if in place like this function name in logs is the right thing to print.
Anyone with access to the code will be able to lookup what function has failed, so maybe some user-level explanation what failed could be better?
For example something like ("Unable to get group cached name '%s'\n", groupname)?
I know that here function name is very straight forward and it name explains everything but it not always has to be true.

Reply:
We have a lot of checks that in theory should never fail.
This is one of examples. 
`sss_get_cased_name()` doesn't look up cached name, but rather makes a copy or copy with chars converted to lower case. I don't think it worth the effort to write explanatory text here.